core/src/Cuda/Kokkos_Cuda_Instance.cpp

```diff
@@ -684,6 +688,30 @@ Cuda::size_type *cuda_internal_scratch_unified(const Cuda &instance,
 
 namespace Kokkos {
 
+bool Cuda::is_running() const {
+  switch (m_space_instance->m_internal_status) {
+    case Impl::ExecutionSpaceStatus::complete: return false;
+    case Impl::ExecutionSpaceStatus::submitted:
+      KOKKOS_IMPL_CUDA_SAFE_CALL(cudaEventRecord(m_space_instance->m_last_event,
+                                                 m_space_instance->m_stream));
+      m_space_instance->m_internal_status = Impl::ExecutionSpaceStatus::running;
+      [[fallthrough]];
+    case Impl::ExecutionSpaceStatus::running:
+      const cudaError_t query_result =
+          cudaEventQuery(m_space_instance->m_last_event);
+      if (query_result == cudaSuccess) {
+        m_space_instance->m_internal_status =
+            Impl::ExecutionSpaceStatus::complete;
+        return false;
+      } else if (query_result != cudaErrorNotReady)
+        KOKKOS_IMPL_CUDA_SAFE_CALL(query_result);
+      return true;
```

Comment:
```suggestion
      {
      const cudaError_t query_result =
          cudaEventQuery(m_space_instance->m_last_event);
      if (query_result == cudaSuccess) {
        m_space_instance->m_internal_status =
            Impl::ExecutionSpaceStatus::complete;
        return false;
      } else if (query_result != cudaErrorNotReady) {
        KOKKOS_IMPL_CUDA_SAFE_CALL(query_result);
      }
      return true;
      }
```
Adding the outer braces to hopefully placate CUDA compiler pickiness about `default:` jumping past initialization of `query_result`, and the inner braces that I asked for on the `else` arm.

Reply:
This would need to be run through `clang-format`

Reply:
See 0cb6c12408eebbae73897ea933416aac8021aee1